web.ml

```diff
@@ -301,7 +301,8 @@ module Http (IO : IO_TYPE) (Curl_IO : CURL with type 'a t = 'a IO.t) : HTTP with
         "url.full", `String url;
       ]
     in
-    let explicit_span = Possibly_otel.enter_manual_span ~__FUNCTION__ ~__FILE__ ~__LINE__ ~data:describe action_name in
+    let explicit_span =
+      Possibly_otel.enter_manual_span ~__FUNCTION__:"Web.Http.add_if_absent" ~__FILE__ ~__LINE__ ~data:describe action_name in
```

Comment:
why Web.Http.add_if_absent?

Reply:
That's what the value `__FUNCTION__` gives there right?

Reply:
Oops actually not, misread, sorry

Reply:
The _real_ value at that location seems to be `"Devkit_core__Web.Http.http_request'.(fun)"`, at least from `dune utop .`, not sure how it shows up in other build contexts.